webapp/src/main/resources/config/application.properties

```diff
@@ -54,9 +57,22 @@ cli.file=mojito-cli.jar
 
 # Quartz
 l10n.org.quartz.scheduler.skipUpdateCheck=true
-l10n.org.quartz.jobStore.useProperties=true
-l10n.org.quartz.scheduler.instanceId=AUTO
-l10n.org.quartz.jobStore.isClustered=true
+
+### for Mysql, cluster deployment
+#l10n.org.quartz.jobStore.useProperties=true
```

Comment:
@jeeyi That's the configuration for Quartz that you need if you deploy a cluster or if you want the job to be retried if the server is restarted in the middle of the job execution. If no setting is added in the conf. it should use in-memory job scheduling and behave like with the old implementation (in memory is not working for a cluster though)